projects/plugins/jetpack/modules/simple-payments/simple-payments.php

```diff
@@ -36,31 +84,56 @@ static function getInstance() {
 		return self::$instance;
 	}
 
+	/**
+	 * Register scripts and styles.
+	 */
 	private function register_scripts_and_styles() {
 		/**
 		 * Paypal heavily discourages putting that script in your own server:
+		 *
 		 * @see https://developer.paypal.com/docs/integration/direct/express-checkout/integration-jsv4/add-paypal-button/
 		 */
-		wp_register_script( 'paypal-checkout-js', 'https://www.paypalobjects.com/api/checkout.js', array(), null, true );
+		wp_register_script(
+			'paypal-checkout-js',
+			'https://www.paypalobjects.com/api/checkout.js',
+			array(),
+			JETPACK__VERSION,
```

Comment:
Does it make sense to pass JETPACK__VERSION as the version for an external script?

Reply:
I'll cc @kraftbj on that one, but it was changed [based on this comment here](https://github.com/Automattic/jetpack/pull/22450#discussion_r804028484).

Reply:
That's a good catch. I think I misread earlier and was thinking the instance below where we were using `self::$version`. Null is fine since this is pulling off of PayPal.

Related, sounds like PayPal is not longer updating the checkout.js approach, wanting us to use their new SDK.

Reply:
Fixed this up in [35314ef](https://github.com/Automattic/jetpack/pull/22450/commits/35314ef067e1641a37e3a22b373368c9f5bc5193)